code/API_definitions/connected-network-type-subscriptions.yaml

```
@@ -535,6 +517,9 @@ components:
         - refreshToken
         - refreshTokenEndpoint
 
+    CreateSubscriptionDetail:
+      description: The detail of the requested event subscription.
+      type: object

```

`CreateSubscriptionDetail` is just a renaming of `SubscriptionDetail`, but we still need to add what subscription details we  require - in this case, the `device`. I made `device` optional as the subscription may be created with a 3-legged token.

The original `SubscriptionDetail` can be deleted, as it is no longer used in the OAS definition
```suggestion
    CreateSubscriptionDetail:
      description: The detail of the requested event subscription.
      type: object
      properties:
        device:
          $ref: "#/components/schemas/Device"
```